charts/harvester-cloud-provider/templates/deployment.yaml

```diff
@@ -48,5 +48,5 @@ spec:
       volumes:
         - name: cloud-config
```

Comment:
Both charts need annotations: https://confluence.suse.com/display/EN/Feature+Charts+Helm+Requirements#FeatureChartsHelmRequirements-Annotations

Reply:
```
  catalog.cattle.io/namespace: cattle-<RESOURCES>-system # Must prefix with cattle- and suffix with -system
```
What is this for?

```
  catalog.cattle.io/ui-component: <FEATURE> # This is added for custom UI deployment of a chart
  catalog.cattle.io/display-name: "Feature Name" # Only for Charts with custom UI
```
Are these two annotations required if we don't have the custom UI?

@aiyengar2

Reply:
`namespace` defines the default namespace we expect to deploy harvester resources within. e.g. for Monitoring V2, we put `cattle-monitoring-system`, for longhorn we put `longhorn-system`.

Yes, I believe we add both annotations regardless of whether or not you need a custom UI. e.g. longhorn uses it despite having a questions.yml UI: https://github.com/rancher/charts/blob/release-v2.6/charts/longhorn/longhorn/100.0.0%2Bup1.1.2/Chart.yaml#L2-L10